shared/actions/chat/inbox.js

```diff
@@ -398,13 +398,19 @@ function* _chatInboxFailedSubSaga(params: RPCChatTypes.ChatUiChatInboxFailedRpcP
   const conversationIDKey = Constants.conversationIDToKey(convID)
 
   // Valid inbox item for rekey errors only
+  let teamname = null
+  if (error.remoteConv.metadata.membersType === RPCChatTypes.commonConversationMembersType.team) {
+    teamname = error.unverifiedTLFName
+  }
   const conversation = Constants.makeInboxState({
     conversationIDKey,
     participants: error.rekeyInfo
       ? I.List([].concat(error.rekeyInfo.writerNames, error.rekeyInfo.readerNames).filter(Boolean))
       : I.List(error.unverifiedTLFName.split(',')),
     status: 'unfiled',
     time: error.remoteConv.readerInfo ? error.remoteConv.readerInfo.mtime : 0,
+    membersType: error.remoteConv.metadata.membersType,
```

Comment:
are there any other fields we should be pulling in that we can?

Reply:
I guess I'll add teamType too